6-AdvancedScenarios/3-call-api-acrs/AppCreationScripts/Configure.ps1

```diff
@@ -193,194 +171,183 @@ Function ConfigureApplications
-                                                  -ReplyUrls "http://localhost:3000" `
-                                                  -PublicClient $False
+    # Create the client AAD application
+    Write-Host "Creating the AAD application (msal-react-spa-acrs)"
+    # create the application 
+    $clientAadApplication = New-AzureADApplication -DisplayName "msal-react-spa-acrs" `
+        -HomePage "http://localhost:3000" `
+        -ReplyUrls "http://localhost:3000" `
+        -PublicClient $False
 
-   # create the service principal of the newly created application 
-   $currentAppId = $clientAadApplication.AppId
-   $clientServicePrincipal = New-AzureADServicePrincipal -AppId $currentAppId -Tags {WindowsAzureActiveDirectoryIntegratedApp}
+    # create the service principal of the newly created application 
+    $currentAppId = $clientAadApplication.AppId
+    $clientServicePrincipal = New-AzureADServicePrincipal -AppId $currentAppId -Tags { WindowsAzureActiveDirectoryIntegratedApp }
 
-   # add the user running the script as an app owner if needed
-   $owner = Get-AzureADApplicationOwner -ObjectId $clientAadApplication.ObjectId
-   if ($owner -eq $null)
-   { 
+    # add the user running the script as an app owner if needed
+    $owner = Get-AzureADApplicationOwner -ObjectId $clientAadApplication.ObjectId
+    if ($owner -eq $null) { 
         Add-AzureADApplicationOwner -ObjectId $clientAadApplication.ObjectId -RefObjectId $user.ObjectId
         Write-Host "'$($user.UserPrincipalName)' added as an application owner to app '$($clientServicePrincipal.DisplayName)'"
-   }
+    }
 
 
-   Write-Host "Done creating the client application (msal-react-spa-acrs)"
+    Write-Host "Done creating the client application (msal-react-spa-acrs)"
 
-   # URL of the AAD application in the Azure portal
-   # Future? $clientPortalUrl = "https://portal.azure.com/#@"+$tenantName+"/blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Overview/appId/"+$clientAadApplication.AppId+"/objectId/"+$clientAadApplication.ObjectId+"/isMSAApp/"
-   $clientPortalUrl = "https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/CallAnAPI/appId/"+$clientAadApplication.AppId+"/objectId/"+$clientAadApplication.ObjectId+"/isMSAApp/"
-   Add-Content -Value "<tr><td>client</td><td>$currentAppId</td><td><a href='$clientPortalUrl'>msal-react-spa-acrs</a></td></tr>" -Path createdApps.html
+    # URL of the AAD application in the Azure portal
+    # Future? $clientPortalUrl = "https://portal.azure.com/#@"+$tenantName+"/blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Overview/appId/"+$clientAadApplication.AppId+"/objectId/"+$clientAadApplication.ObjectId+"/isMSAApp/"
+    $clientPortalUrl = "https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/CallAnAPI/appId/" + $clientAadApplication.AppId + "/objectId/" + $clientAadApplication.ObjectId + "/isMSAApp/"
```

Comment:
>$clientPortalUrl [](http://example.com/codeflow?start=4&length=16)

the earlier code used to print this url on console when manual steps were printed. This used to make the job of locating the app much easier in the portal. Could you check what happened..